firefox-ios/Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -437,7 +437,11 @@ class FakespotViewController: UIViewController,
             return view
 
         case .highlightsCard:
-            guard let cardViewModel = viewModel.highlightsCardViewModel else { return nil }
+            guard var cardViewModel = viewModel.highlightsCardViewModel else { return nil }
+            cardViewModel.expandState = fakespotState.isHighlightsSectionExpanded ? .expanded : .collapsed
+            cardViewModel.onExpandStateChanged = { state in
```

Comment:
I don't think we need the closure here. The `FakespotHighlightsCardView` can call `store.dispatch()` itself.

Reply:
I kept the same pattern even `FakespotHighlightsCardView` is not an expandable card for consistency. In theory we can access and send actions to the store from everywhere but I imagine the code becoming messier and harder to change in the future.